install_config/configuring_local.adoc

```diff
@@ -0,0 +1,180 @@
+[[install-config-configuring-local]]
+= Configuring for Local Volume
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+{product-title} can be configured to access
+xref:../install_config/persistent_storage/persistent_storage_local.adoc#install-config-persistent-storage-persistent-storage-local[Local
+Volumes] for application data.
+
+Local volumes are PersistentVolumes(PV) representing local mounted filesystems. In future, they may be extended to raw block devices.
+
+Local volumes are different from HostPath, they have a special annotation that makes any Pod that uses the PV, to be scheduled on the same node where the local volume is mounted.
```

Comment:
s/HostPath, / HostPath. (period not comma). 
They have a special annotation  (new sentence)
Remove comma after PV